frontend/src/features/book/BookDetailSection.tsx

```diff
@@ -1,30 +1,34 @@
+import EmptyUI from "@/components/EmptyUI";
 import Table from "@/components/table/Table";
+import type { BookData } from "@/features/book/Book.types";
 import RouteCard from "@/features/book/RouteCard";
 import { theme } from "@/styles/themes.style";
-import type { TableObject } from "@/utils/TableMatcher";
+import TableMatcher from "@/utils/TableMatcher";
 import TabMatcher from "@/utils/TabMatcher";
 import { css } from "@emotion/react";
 const { typography } = theme;
 
 interface BookDetailSectionProps {
+  data: BookData | undefined;
   activeTab: string;
-  userRows: TableObject[];
-  bookingRows: TableObject[];
-  routeRows: TableObject[];
 }
 
-const BookDetailSection = ({
-  activeTab,
-  userRows,
-  bookingRows,
-  routeRows,
-}: BookDetailSectionProps) => {
+const BookDetailSection = ({ data, activeTab }: BookDetailSectionProps) => {
+  // tab 로직
+  const parsedActiveTab = TabMatcher.matchBookTypeKRToENG(activeTab);
+
+  // table 로직
+  const { id, bookStatus, ...activeBook } = data ?? {}; //id, bookStatus 제외
```

Comment:
사용하지 않는데, 구조분해를 위해 선언해둔 변수들은 _를 이용하면 린트 에러를 피할 수 있습니다 !
```suggestion
  const { id: _id, bookStatus: _bookStatus, ...activeBook } = data ?? {};
```